src/test/java/io/appium/java_client/service/local/ServerBuilderTest.java

```diff
@@ -114,6 +115,15 @@ public void tearDown() throws Exception {
         ofNullable(PATH_TO_APPIUM_NODE_IN_PROPERTIES).ifPresent(s -> setProperty(APPIUM_PATH, s));
     }
 
+    @Test public void checkAbilityToAddLogMessageConsumer() {
+        List<String> log = new ArrayList<>();
+        service = buildDefaultService();
+        service.clearOutPutStreams();
+        service.addLogMessageConsumer(log::add);
```

Comment:
should we also have a possibility to remove a consumer?

Reply:
Essentially a consumer in this case is an OutputStream (as added with `addOutPutStream(...)` method). So consumers can be removed with the `clearOutPutStreams()` method.